### ISSUE TITLE
Undefined array key "Stop"

### ISSUE THREAD

**Issue by niclashedam** [lsv/rejseplan-php-api#12]
Requesting the JourneyDetails of a DepartureBoardData or URL may result in Stop being undefined. I am using version 4.0 and Guzzle 7.4.1 as client.

```
$journey = (new Journey("http://webapp.rejseplanen.dk/bin//rest.exe/journeyDetail?ref=392787%2F133259%2F725430%2F231787%2F86%3Fdate%3D25.01.22%26format%3Djson"))->request();

```
```
  Undefined array key "Stop"

  at vendor/lsv/rejseplan-php-api/src/Journey.php:50
     46▕
     47▕     protected function makeResponse(string $response): string
     48▕     {
     49▕         $details = json_decode($response, true, flags: JSON_THROW_ON_ERROR)['JourneyDetail'];
  ➜  50▕ 	 $stops = $details['Stop'];
     51▕         if (isset($stops['name'])) {
     52▕             $stops = [$stops];
     53▕         }
     54▕
```

$response contains the following
```

{
"JourneyDetail":{
  "noNamespaceSchemaLocation":"http://webapp.rejseplanen.dk/xml/rest/hafasRestJourneyDetail.xsd",
  "error":"C3 cgiError",
  "$":""
  }
}
```

The data from the API looks like this.

```
{
"JourneyDetail":{
  "noNamespaceSchemaLocation":"http://webapp.rejseplanen.dk/xml/rest/hafasRestJourneyDetail.xsd",
  "Stop":[{
    "name":"Aarhus H",
    "x":"10204761",
    "y":"56150444",
    "routeIdx":"0",
    "depTime":"10:58",
    "depDate":"25.01.22",
    "track":"6",
    "$":""
    },{
    "name":"Viby Jylland St.",
    "x":"10160948",
    "y":"56122353",
    "routeIdx":"1",
    "arrTime":"11:02",
    "arrDate":"25.01.22",
    "depTime":"11:03",
    "depDate":"25.01.22",
    "track":"4",
    "$":""
    },{
    "name":"Hørning St.",
    "x":"10037787",
    "y":"56082046",
    "routeIdx":"2",
    "$":""
    },{
    "name":"Skanderborg St.",
    "x":"9925359",
    "y":"56043374",
    "routeIdx":"3",
    "arrTime":"11:13",
    "arrDate":"25.01.22",
    "depTime":"11:14",
    "depDate":"25.01.22",
    "track":"2",
    "$":""
    },{
    "name":"Horsens St.",
    "x":"9836536",
    "y":"55862763",
    "routeIdx":"4",
    "arrTime":"11:27",
    "arrDate":"25.01.22",
    "depTime":"11:29",
    "depDate":"25.01.22",
    "track":"3",
    "$":""
    },{
    "name":"Hedensted St.",
    "x":"9698462",
    "y":"55767567",
    "routeIdx":"5",
    "arrTime":"11:37",
    "arrDate":"25.01.22",
    "depTime":"11:38",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Vejle St.",
    "x":"9536485",
    "y":"55706737",
    "routeIdx":"6",
    "arrTime":"11:47",
    "arrDate":"25.01.22",
    "depTime":"11:49",
    "depDate":"25.01.22",
    "track":"4",
    "$":""
    },{
    "name":"Brejning St.",
    "x":"9659278",
    "y":"55666510",
    "routeIdx":"7",
    "arrTime":"11:55",
    "arrDate":"25.01.22",
    "depTime":"11:56",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Børkop St.",
    "x":"9651943",
    "y":"55642015",
    "routeIdx":"8",
    "arrTime":"11:59",
    "arrDate":"25.01.22",
    "depTime":"12:00",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Fredericia St.",
    "x":"9740639",
    "y":"55568213",
    "routeIdx":"9",
    "arrTime":"12:08",
    "arrDate":"25.01.22",
    "depTime":"12:13",
    "depDate":"25.01.22",
    "track":"7",
    "$":""
    },{
    "name":"Taulov St.",
    "x":"9615761",
    "y":"55545120",
    "routeIdx":"10",
    "arrTime":"12:19",
    "arrDate":"25.01.22",
    "depTime":"12:20",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Kolding St.",
    "x":"9481094",
    "y":"55490834",
    "routeIdx":"11",
    "arrTime":"12:26",
    "arrDate":"25.01.22",
    "depTime":"12:28",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Lunderskov St.",
    "x":"9303377",
    "y":"55479975",
    "routeIdx":"12",
    "arrTime":"12:35",
    "arrDate":"25.01.22",
    "depTime":"12:36",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Vejen St.",
    "x":"9136187",
    "y":"55475939",
    "routeIdx":"13",
    "arrTime":"12:43",
    "arrDate":"25.01.22",
    "depTime":"12:44",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Brørup St.",
    "x":"9016136",
    "y":"55482798",
    "routeIdx":"14",
    "arrTime":"12:50",
    "arrDate":"25.01.22",
    "depTime":"12:51",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Holsted St.",
    "x":"8914459",
    "y":"55484928",
    "routeIdx":"15",
    "arrTime":"12:56",
    "arrDate":"25.01.22",
    "depTime":"12:57",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Gørding St.",
    "x":"8800727",
    "y":"55469700",
    "routeIdx":"16",
    "arrTime":"13:02",
    "arrDate":"25.01.22",
    "depTime":"13:02",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Bramming St.",
    "x":"8706799",
    "y":"55464774",
    "routeIdx":"17",
    "arrTime":"13:06",
    "arrDate":"25.01.22",
    "depTime":"13:07",
    "depDate":"25.01.22",
    "track":"1",
    "$":""
    },{
    "name":"Tjæreborg St.",
    "x":"8579368",
    "y":"55466392",
    "routeIdx":"18",
    "$":""
    },{
    "name":"Jerne St.",
    "x":"8485835",
    "y":"55474537",
    "routeIdx":"19",
    "$":""
    },{
    "name":"Esbjerg St.",
    "x":"8458193",
    "y":"55467920",
    "routeIdx":"20",
    "arrTime":"13:18",
    "arrDate":"25.01.22",
    "track":"2",
    "$":""
    }],
  "JourneyName":{
    "name":"Re 3334",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    },
  "JourneyType":{
    "type":"REG",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    },
  "Note":[{
    "text":"Retning: Esbjerg St.",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    },{
    "text":"kører:daglig, ikke 12., 13. feb, 26., 27. mar",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    },{
    "text":"Toget har lav indstigning",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    },{
    "text":"Mulighed for internet",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    },{
    "text":"DSB 1' Regional",
    "routeIdxFrom":"0",
    "routeIdxTo":"20"
    }],
  "MessageList":{
    "Message":{
      "Header":{
        "$":"\nHusk mundbind eller visir\n"
        },
      "Text":{
        "$":"\nDer er krav om mundbind eller visir på stationer, busterminaler og i al kollektiv trafik\n"
        }
      }
    }
  }
}
```

**Comment by lsv** [comment]
Thanks, ill look it up and see why this happens :)

**Comment by lsv** [comment]
It has now been solved in 4.0.1 thanks @niclashedam
